src/views/Pipeline.vue

```diff
@@ -457,21 +457,18 @@ export default defineComponent({
         event.target.complete();
       })
     },
-    async refreshJobs(event: any) {
-      this.isRetrying = true;
+    async refreshJobs(event: any, isRetrying = false ) {
+      this.isRetrying = isRetrying;
       if(this.segmentSelected === 'pending') {
         this.getPendingJobs().then(() => {
-          if(event) event.target.complete();
```

Comment:
Check if it is not breaking pull to refresh (ion-refresher) 
https://ionicframework.com/docs/api/refresher#complete

Reply:
I tried "pull to refresh" on Pipeline page & loader keeps on spinning.

Reply:
We could pass event as undefined for all the other cases then pull to refresh

Reply:
Done sir.